.travis.yml

```diff
@@ -26,6 +27,8 @@ env:
     - PYTHON_DEBUG=1
     - WINEDEBUG=fixme-all
   matrix:
+# bitcoind
```

Comment:
I know that its not necessary, but I think that its generally useful so that we get the full test results ASAP rather than waiting for the crossbuilds which are a lot less likely to fail.